### PR TITLE
Question about sorting algorithms

### PR DESCRIPTION
**Pull request by sotoblanco**
This question allows evaluation of the understanding of why sorting algorithms are important, which might increase the ability of the student to learn it.

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/general-cs-quiz.ts [hunk 1]
```diff
@@ -1430,6 +1430,15 @@ const generalCSQuiz = [
     Distractor3: "O(log n)" ,
     Explanation: "The letter n represents the input size, and the big O represents the increase of the running time." ,
     Link: "https://www.freecodecamp.org/news/big-o-cheat-sheet-time-complexity-chart/"
+   },
+  {
+    Question: "Why Sorting Algorithms are Important?" ,
```
Comment (jdwilkin4) [inline src/data/general-cs-quiz.ts:1435]:
```suggestion
    Question: "Why are sorting algorithms important in programming?" ,
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by sotoblanco** [comment]
Thank you

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**